examples/basecoin/glide.lock

```diff
@@ -0,0 +1,151 @@
+hash: 74ba16fcb7dac2ceca406dcc6c6ed00ab1cd09af4ca6f3f1d35452254e149fb4
```

Comment:
Why do we have this glide.lock here?
Isn't it better just to use the glide.lock in the root directory.
It makes it very hard to upgrade and test code with basecoin otherwise, especially as the repo is in a large state of flux now.